dnf5/context.cpp

```diff
@@ -950,8 +949,8 @@ std::vector<std::string> match_specs(
     base.load_config_from_file();
     base.setup();
 
-    // optimization - if pattern contain '/', disable the search for matching installed and available packages
-    if (pattern.find('/') != std::string::npos) {
+    // optimization - disable the search for matching installed and available packages for file patterns
+    if (libdnf::utils::is_file_pattern(pattern)) {
```

Comment:
I thing the change is correct, but the description of function is not sufficient. I got wrong impression what it does. Internally it creates another instance of Base() and then it suggest argument name for auto-compleation.

Reply:
I just changed the comment for the `is_file_pattern` call here to match the current reference. But you probably mean the description of the `match_specs` function? Should I updated the doc string for it in the `context.hpp`?

Reply:
I am sorry, I wasn't explicit. I meant the description of `match_specs`. If someone will use that method for something else then callback of an argument parser => it will be not nice.

Reply:
If you can also modify `match_specs` description it would be great.

Reply:
I've updated the docstring in the context.hpp. But anyway for me it seems this is not the right place for this function. It is sticked to the context file, but is static and related only to the argument parser...